spark/v3.3/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestSetWriteDistributionAndOrdering.java

```diff
@@ -64,6 +67,25 @@ public void testSetWriteOrderByColumn() {
     Assert.assertEquals("Should have expected order", expected, table.sortOrder());
   }
 
+  @Test
+  public void testSetWriteOrderWithCaseSensitiveColumnNames() {
+    sql(
+        "CREATE TABLE %s (Id bigint NOT NULL, Category string, ts timestamp, data string) USING iceberg",
+        tableName);
+    Table table = validationCatalog.loadTable(tableIdent);
+    Assert.assertTrue("Table should start unsorted", table.sortOrder().isUnsorted());
+    Assertions.assertThatThrownBy(() -> {
+      sql(String.format("SET %s=true", SQLConf.CASE_SENSITIVE().key()));
```

Comment:
Doesn't sql() already do format?

Reply:
Should we also move the set outside?  So that assertThatThrownBy is only about the alter stmt?

Reply:
Fixed the `sql` and move the set commands outside